resources/js/services/authService.js

```diff
@@ -51,3 +51,11 @@ export const logout = async (token) => {
       store.dispatch(logoutAction());
     });
 };
+
+export const accountCreation = async (token,fullname, address) => {
+  const response = await getAxiosInstance(token)
+    .post('createAccount',{fullname,address});
+
+  return response.data;
+  // console.log(response);
```

Comment:
@raghuhvarman Let's remove the logs completely if possible.

Reply:
Fixed